rocketpy/motors/HybridMotor.py

```diff
@@ -0,0 +1,526 @@
+        Motor.massDot : Function
+            Time derivative of propellant total mass in kg/s as a function
+            of time.
+        Motor.inertiaI : Function
+            Propellant moment of inertia in kg*meter^2 with respect to axis
+            perpendicular to axis of cylindrical symmetry of each grain,
+            given as a function of time.
+        Motor.inertiaIDot : Function
+            Time derivative of inertiaI given in kg*meter^2/s as a function
+            of time.
+        Motor.inertiaZ : Function
+            Propellant moment of inertia in kg*meter^2 with respect to axis of
+            cylindrical symmetry of each grain, given as a function of time.
+        Motor.inertiaDot : Function
+            Time derivative of inertiaZ given in kg*meter^2/s as a function
+            of time.
+
+        Thrust and burn attributes:
+        Motor.thrust : Function
+            Motor thrust force, in Newtons, as a function of time.
+        Motor.totalImpulse : float
+            Total impulse of the thrust curve in N*s.
+        Motor.maxThrust : float
+            Maximum thrust value of the given thrust curve, in N.
+        Motor.maxThrustTime : float
+            Time, in seconds, in which the maximum thrust value is achieved.
+        Motor.averageThrust : float
+            Average thrust of the motor, given in N.
+        Motor.burnOutTime : float
+            Total motor burn out time, in seconds. Must include delay time
+            when the motor takes time to ignite. Also seen as time to end thrust
+            curve.
+        Motor.exhaustVelocity : float
+            Propulsion gases exhaust velocity, assumed constant, in m/s.
+        Motor.burnArea : Function
+            Total burn area considering all grains, made out of inner
+            cylindrical burn area and grain top and bottom faces. Expressed
+            in meters squared as a function of time.
+        Motor.Kn : Function
+            Motor Kn as a function of time. Defined as burnArea divided by
+            nozzle throat cross sectional area. Has no units.
+        Motor.burnRate : Function
+            Propellant burn rate in meter/second as a function of time.
+        Motor.interpolate : string
+            Method of interpolation used in case thrust curve is given
+            by data set in .csv or .eng, or as an array. Options are 'spline'
+            'akima' and 'linear'. Default is "linear".
+    """
+
+    def __init__(
+        self,
+        thrustSource,
+        burnOut,
+        grainsCenterOfMassPosition,
+        grainNumber,
+        grainDensity,
+        grainOuterRadius,
+        grainInitialInnerRadius,
+        grainInitialHeight,
+        grainSeparation=0,
+        nozzleRadius=0.0335,
+        nozzlePosition=0,
+        throatRadius=0.0114,
+        reshapeThrustCurve=False,
+        interpolationMethod="linear",
+        coordinateSystemOrientation="nozzleToCombustionChamber",
+    ):
```

Comment:
What is up with the optional argument values of `nozzleRadius` and `throatRadius`? I know this comes from SolidMotor but I think it is time to change it.

Reply:
In fact, we had already fixed this in the `SolidMotor` class. I'll update the `HybridMotor` constructor to follow the same pattern: no default value for `grainSeparation` and `nozzleRadius`. The `throatRadius` keeps a default value of `0.01` due to its importance only for computing Kn.